localstack-core/localstack/services/events/utils.py

```diff
@@ -38,6 +38,10 @@ def default(self, obj):
         return super().default(obj)
 
 
+def to_json(obj: Any, separators: tuple[str] = None) -> str:
+    return json.dumps(obj, separators, cls=EventJSONEncoder)
```

Comment:
Separators are a positional argument: https://docs.python.org/3/library/json.html#json.JSONEncoder

This fix should resolve the failing EventBridge v2 provider tests

Reply:
You can check out this sample from Marco in SFN: `localstack.services.stepfunctions.asl.utils.encoding.to_json_str`

Reply:
Done thanks for the hint :)